src/lib/packager/clients/pkg_finish.rb

```diff
@@ -0,0 +1,198 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# ------------------------------------------------------------------------------
+
+require "installation/finish_client"
+require "packages/repository"
+
+module Yast
+  class PkgFinishClient < ::Installation::FinishClient
+    include Yast::I18n
+    include Yast::Logger
+
+    # Path to libzypp repositories
+    REPOS_DIR = "/etc/zypp/repos.d"
+    # Repository schemes considered local (see disable_local_repos)
+    # https://github.com/openSUSE/libzypp/blob/a7a038aeda1ad6d9e441e7d3755612aa83320dce/zypp/Url.cc#L458
+    LOCAL_SCHEMES = [:cd, :dvd, :dir, :hd, :iso, :file]
```

Comment:
That's the wrong approach. I'll move this logic to a `Packages::Repository#local?` method.